### DISCUSSION TITLE
Order list menu to reflect chosen sort order instead of always by recent

### DISCUSSION THREAD

**Discussion by blond-in-blue** [community/community#13176] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I love the lists feature and have a suggestion to make it easier to use. When adding things to lists, the items on the list menu jump around by adding whatever was recently used to the top. I appreciate that selected items go to the top of the menu, but when adding new things to different lists the automatic reordering can quickly get confusing as the list order is constantly jumbled around. My suggestion is for the menu to reflect whatever method the user has selected to sort their lists in the list menu as well. 

Thank you!

![List Menu Example 1](https://user-images.githubusercontent.com/6818170/159368466-a392f37d-979e-44b6-ab26-3373d7e05f1e.png)
![List Menu Example 2](https://user-images.githubusercontent.com/6818170/159368464-9734a268-88a8-4ef2-9357-89cb491e2a31.png)
![List Menu Example 3](https://user-images.githubusercontent.com/6818170/159368463-2aa2d8c6-1321-4c97-b454-856198f5001e.png)
![List Sort Options](https://user-images.githubusercontent.com/6818170/159368461-cb628b82-4e0e-4fb3-8d07-fb7cd828173a.png)

**Comment by jsoref** [comment, marked as answer]
Covered by https://github.com/github/feedback/discussions/9094
